src/app/clusters/door-lock-server/door-lock-server.cpp

```diff
@@ -2085,8 +2085,10 @@ DlStatus DoorLockServer::addCredentialToUser(chip::EndpointId endpointId, chip::
 
     for (size_t i = 0; i < user.credentials.size(); ++i)
     {
-        // appclusters, 5.2.4.40: user should not be already associated with given credentialIndex
```

Comment:
You probably want to keep the old spec reference too, no?  That's where the fact that we're doing this check at all is defined.

Reply:
Yes, that is a good point. I will add the reference.